### ISSUE TITLE
Header/Footer: Determine External link visual and behavior

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#16202]
## Description

We need to determine how external links should behave and look in the Federal header and footer. According to the [external link guidance](https://design.va.gov/content-style-guide/links#linking-to-external-sites) in the VA Design system:
- All External links should open in a new tab
- Any VA subdomains on the exception links should have "(opens in a new tab)"
  - It's possible we can replace this text with the external link icon
 
However, this guidance seems unfinished. For accessibility, we should always indicate if a link is opening in a new tab, not just these exceptions. I also think we want to confirm if there are are links within the new header/footer that should be exceptions in this sub-domain list. 

## Acceptance Criteria
- [ ] Laura and Amanda to discuss with stakeholders on 1/8
- [ ] Testable_Outcome_Y
- [ ] Testable_Outcome_Z

## Team
Please check the team(s) that will do this work.

- [ ] CMS Team
- [x] Public Websites
- [ ] Facilities
- [ ] Accelerated Publishing

**Comment by FranECross** [comment]
@FranECross and @laflannery to work on AC/Descip on Thurs 12/14.

**Comment by FranECross** [comment]
Pending Dave getting information from others. Not an implementation blocker.

**Comment by laflannery** [comment]
@FranECross @jilladams My handoff notes to the DST included an ask to determine and document external link behavior. Given the new direction for the Header/Footer project, I would imagine this particular ticket should be closed now as our team is not going to be updating this?

**Comment by laflannery** [comment]
Does this thumbs up mean I can close this?

**Comment by FranECross** [comment]
@laflannery Sorry, yes that's a thumbs up okay to close. I'll close!
